### PR TITLE
tor status UI update

### PR DESCRIPTION
**Pull request by nikhilrayaprolu**
in reference to issue #437 
added a preloader type status update 
reference used: http://www.w3schools.com/howto/howto_css_loader.asp
screenshot - 22 07 2016 - 3 38 50
screenshot - 22 07 2016 - 3 39 08

### CODE REVIEW

File: plinth/modules/tor/templates/tor.html [hunk 1]
```diff
@@ -26,22 +26,33 @@
...
 {% block configuration %}
 
-  <h3>{% trans "Status" %}</h3>
+  
 
   {% if config_running %}
-
-    <p class="running-status-parent">
-      <span class="running-status active"></span>
-      {% trans "Tor configuration is being updated" %}
```
Comment (jvalleroy) [inline plinth/modules/tor/templates/tor.html]: I think we should keep this text. Is there a reason to remove it?

File: plinth/modules/tor/templates/tor.html [hunk 2]
```diff
@@ -26,22 +26,33 @@
...
+    0% { transform: rotate(0deg); }
+    100% { transform: rotate(360deg); }
+}
+</style>
 {% endblock %}
 
 
 {% block configuration %}
 
-  <h3>{% trans "Status" %}</h3>
```
Comment (jvalleroy) [inline plinth/modules/tor/templates/tor.html]: Is there a reason to remove "Status" here and add it in 2 places below? It seems like there would be no difference in the final display.

**Comment by nikhilrayaprolu** [comment]
@jvalleroy  I modified it keeping back the removed lines
screenshot - 23 07 2016 - 11 36 54

**Comment by jvalleroy** [comment]
I've merged this change, and made some minor updates to the template style.

**Pull request closed without merging**